<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyCalculator</class>
 <widget class="QMainWindow" name="MyCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyCalculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#widget{
  background-color: #FFF7D1;
  border-radius: 10px;
}

#multi, #moins, #plus, #equal, #clear, #divise{
	background-color: #DF9755; 
    padding : 10px; 
    border-radius: 10px; 
	font-size: 17px;
	font-weight: 500; 
    color : #000;
	text-align: center; 

}
#nb_1, #nb_2, #nb_3, #nb_4, #nb_5, #nb_6, #nb_8, #nb_7, #nb_9, #nb_0{
	background-color: #3D3D3D;
	padding : 10px; 
    border-radius: 10px; 
	font-size: 15px;
}

#frame_2{
  padding : 5px;
  width: 500px;
  border:none;
}

#frame{
  border: none;
}

QLineEdit{
   border-radius: 10px; 
   border:none;
   padding : 50px;
   background-color: #FFF7D1;
   margin-bottom: 30px;
   color:#000;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignmentFlag::AlignTop">
        <widget class="QWidget" name="scream" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignmentFlag::AlignTop">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="oparation" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>-1</number>
             </property>
             <item row="0" column="0">
              <widget class="QPushButton" name="nb_7">
               <property name="text">
                <string>7</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="nb_8">
               <property name="text">
                <string>8</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="nb_9">
               <property name="text">
                <string>9</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="nb_4">
               <property name="text">
                <string>4</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="nb_5">
               <property name="text">
                <string>5</string>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QPushButton" name="multi">
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <fontweight>Medium</fontweight>
                </font>
               </property>
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="nb_1">
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="nb_2">
               <property name="text">
                <string>2</string>
               </property>
              </widget>
             </item>
             <item row="2" column="4">
              <widget class="QPushButton" name="moins">
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <fontweight>Medium</fontweight>
                </font>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="nb_0">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="clear">
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <fontweight>Medium</fontweight>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>C</string>
               </property>
              </widget>
             </item>
             <item row="3" column="4">
              <widget class="QPushButton" name="plus">
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <fontweight>Medium</fontweight>
                </font>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QPushButton" name="divise">
               <property name="text">
                <string>รท </string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="nb_6">
               <property name="text">
                <string>6</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QPushButton" name="nb_3">
               <property name="text">
                <string>3</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QPushButton" name="equal">
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <fontweight>Medium</fontweight>
                </font>
               </property>
               <property name="text">
                <string>=</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
